src/pages/manage/movies/MovieForm.tsx

```diff
@@ -309,16 +332,12 @@ const MovieForm = () => {
 
   const createNewEvent = async ({
     production,
-    cinema,
-    theme: themeId,
+    place,
+    eventTypeAndTheme: { eventType, theme },
```

Comment:
imho I think `eventCategoryAndTheme` would be a better name.
eventType confuses me with offerType...

Reply:
EventType is the actual naming though

https://publiq.stoplight.io/docs/uitdatabank/ZG9jOjUwNjExMQ-taxonomy-terms

Reply:
<img width="549" alt="Screenshot 2022-03-10 at 15 00 29" src="https://user-images.githubusercontent.com/89046/157677498-d99fe2e8-33bd-4819-a955-929a8e78e46f.png">

Reply:
👍